package.json

```diff
@@ -44,7 +44,7 @@
 	"dependencies": {
 		"cross-spawn": "^7.0.3",
 		"get-stream": "^6.0.1",
```

Comment:
Upgrading `get-stream` is not as straightforward, so I'll do it in a separate PR. This probably should be done before making the major release.